src/Assets/Elements/StyledButton/index.js

```diff
@@ -0,0 +1,14 @@
+import { Button } from '@material-ui/core';
+import styled from 'styled-components';
+
+export const StyledButton = styled(Button)`
```

Comment:
is this the default style for a button in all the screens?
If yes let's notify all the developers about this component

Reply:
Idk if this is the default button for everyone it is for me. we need to discuss about it if they are using something else and if we can combine it to one component